open_source/gravwell-collectd-ingester-licenses/licenses.txt

```diff
@@ -1,4 +1,4 @@
-collectd.org,https://github.com/collectd/go-collectd/blob/v0.5.0/LICENSE,ISC
+collectd.org,Unknown,ISC
```

Comment:
When I tried to go to the website, the certificate is expired. I think that's why this is currently showing as Unknown